timeless/access_control/owner_privileges.py

```diff
@@ -25,6 +25,19 @@ def __location_access(method=None, *args):
     return permitted
 
 
+def __employee_access(method=None, *args, **kwargs):
+    permitted = False
+    user = g.pop("user", None)
+    employee_id = kwargs.get("employee_id")
+    if method == Method.READ and user:
+        if not employee_id:
+            permitted = True
+        else:
+            permitted = method == Method.READ and employee_id == user.id
```

Comment:
@Vatavuk I wrote shorter:

```python
def __employee_access(method=None, employee_id=None, *args, **kwargs):
    permitted, user = False, g.get("user")

    if method == Method.READ and user:
        if employee_id:
            return employee_id == user.id
        return True
    return permitted
```
but now I'm not sure about the logic. is it fine that if no `employee_id` provided than everyone can read?

Reply:
@vryazanov I don't like multiple return statements. Will combine your logic with one return statement. Regarding privlieges this just test if the user can access endpoint `list` when no `employeeId` is provided. The other logic which returns his own employees should be done elswhere.

Reply:
@Vatavuk got it. what about 
```python
def __employee_access(method=None, employee_id=None, *args, **kwargs):
    permitted, user = False, g.get("user")

    if method == Method.READ and user:
        if employee_id:
            permitted = employee_id == user.id
        else:
            permitted = True
    return permitted
```
Looks a bit more understandable. what do you think?

Reply:
@vryazanov 👍